src/test/java/io/confluent/examples/streams/SumLambdaIntegrationTest.java

```diff
@@ -89,33 +79,29 @@ public void shouldSumEvenNumbers() throws Exception {
         .reduce((v1, v2) -> v1 + v2);
     sumOfOddNumbers.toStream().to(outputTopic);
 
-    final KafkaStreams streams = new KafkaStreams(builder.build(), streamsConfiguration);
-    streams.start();
+    final TopologyTestDriver topologyTestDriver = new TopologyTestDriver(builder.build(), streamsConfiguration);
```

Comment:
I wonder would it be worth at the same time get rid of this copy paste code in this Test class and utilize the topology directly from SumLambdaExample.java

Reply:
Oh, wow. I didn't even notice that the whole topology is just duplicated. Kind of embarrassing, actually. Thanks for the catch!

Reply:
So, I did this and learned that the test was summing *even* numbers, while the example was summing *odd* numbers. Oops!

It's fixed now.